[Bug]: There is no need to replace some functions in `KryptonCheckButtonActionList`

**Describe the bug**
``` c#
1>Z:\GitHub\Krypton-Suite\Standard-Toolkit\Source\Krypton Components\Krypton.Toolkit\Designers\Action Lists\KryptonCheckButtonActionList.cs(78,33,78,49): warning CS0108: 'KryptonCheckButtonActionList.ContextMenuStrip' hides inherited member 'KryptonButtonActionList.ContextMenuStrip'. Use the new keyword if hiding was intended.
1>Z:\GitHub\Krypton-Suite\Standard-Toolkit\Source\Krypton Components\Krypton.Toolkit\Designers\Action Lists\KryptonCheckButtonActionList.cs(95,21,95,34): warning CS0108: 'KryptonCheckButtonActionList.ShortTextFont' hides inherited member 'KryptonButtonActionList.ShortTextFont'. Use the new keyword if hiding was intended.
1>Z:\GitHub\Krypton-Suite\Standard-Toolkit\Source\Krypton Components\Krypton.Toolkit\Designers\Action Lists\KryptonCheckButtonActionList.cs(112,21,112,33): warning CS0108: 'KryptonCheckButtonActionList.LongTextFont' hides inherited member 'KryptonButtonActionList.LongTextFont'. Use the new keyword if hiding was intended.
1>Z:\GitHub\Krypton-Suite\Standard-Toolkit\Source\Krypton Components\Krypton.Toolkit\Designers\Action Lists\KryptonCheckButtonActionList.cs(130,20,130,32): warning CS0108: 'KryptonCheckButtonActionList.CornerRadius' hides inherited member 'KryptonButtonActionList.CornerRadius'. Use the new keyword if hiding was intended.
```

**To Reproduce**
Steps to reproduce the behaviour:
Build the code !


**Expected behavior**
The `KryptonCheckButton` is derived from the `KryptonButton`, so replacing these functions just complicates the build and the expectation so of the code

**Screenshots**
If applicable, add screenshots to help explain your problem.

**Desktop (please complete the following information):**
 - OS: [e.g. Windows 10]
 - Version 6.0.0.0

--- a/README.md
+++ b/README.md
@@ -109,6 +109,7 @@ These packages only support .NET Framework 4.8 and .NET 5/6.
 * New `KryptonMessageBoxManager`, now you can configure a `KryptonMessageBox` through the designer
 * Fixed [#106](https://github.com/Krypton-Suite/Standard-Toolkit/issues/106), 'MethodImplOptions' does not contain a definition for 'AggressiveInlining'
 * Fixed [#102](https://github.com/Krypton-Suite/Standard-Toolkit/issues/102), `KryptonInputBox` does not compile
+* Fixed [#99](https://github.com/Krypton-Suite/Standard-Toolkit/issues/99), There is no need to replace some functions in `KryptonCheckButtonActionList`
 * Documentation [#97](https://github.com/Krypton-Suite/Standard-Toolkit/issues/97), Add descriptions & documentation
 * Implemented [#93](https://github.com/Krypton-Suite/Standard-Toolkit/issues/93), For a `KryptonMessageBox`: Please describe difference between **owner** and **parentWindow** parameters
 * Fixed [#92](https://github.com/Krypton-Suite/Standard-Toolkit/issues/92), **Many* Items cannot be Dragged and dropped onto a Form in Designer

--- a/Source/Krypton Components/Krypton.Toolkit/Designers/Action Lists/KryptonCheckButtonActionList.cs
+++ b/Source/Krypton Components/Krypton.Toolkit/Designers/Action Lists/KryptonCheckButtonActionList.cs
@@ -75,79 +75,6 @@ namespace Krypton.Toolkit
                 }
             }
         }
-
-        /// <summary>Gets or sets the context menu strip.</summary>
-        /// <value>The context menu strip.</value>
-        // TODO: https://github.com/Krypton-Suite/Standard-Toolkit/issues/99
-        public ContextMenuStrip ContextMenuStrip
-        {
-            get => _checkButton.ContextMenuStrip;
-
-            set
-            {
-                if (_checkButton.ContextMenuStrip != value)
-                {
-                    _service.OnComponentChanged(_checkButton, null, _checkButton.ContextMenuStrip, value);
-
-                    _checkButton.ContextMenuStrip = value;
-                }
-            }
-        }
-
-        /// <summary>Gets or sets the font.</summary>
-        /// <value>The font.</value>
-        // TODO: https://github.com/Krypton-Suite/Standard-Toolkit/issues/99
-        public Font ShortTextFont
-        {
-            get => _checkButton.StateCommon.Content.ShortText.Font;
-
-            set
-            {
-                if (_checkButton.StateCommon.Content.ShortText.Font != value)
-                {
-                    _service.OnComponentChanged(_checkButton, null, _checkButton.StateCommon.Content.ShortText.Font, value);
-
-                    _checkButton.StateCommon.Content.ShortText.Font = value;
-                }
-            }
-        }
-
-        /// <summary>Gets or sets the font.</summary>
-        /// <value>The font.</value>
-        // TODO: https://github.com/Krypton-Suite/Standard-Toolkit/issues/99
-        public Font LongTextFont
-        {
-            get => _checkButton.StateCommon.Content.LongText.Font;
-
-            set
-            {
-                if (_checkButton.StateCommon.Content.LongText.Font != value)
-                {
-                    _service.OnComponentChanged(_checkButton, null, _checkButton.StateCommon.Content.LongText.Font, value);
-
-                    _checkButton.StateCommon.Content.LongText.Font = value;
-                }
-            }
-        }
-
-        /// <summary>Gets or sets the corner radius.</summary>
-        /// <value>The corner radius.</value>
-        [DefaultValue(-1)]
-        // TODO: https://github.com/Krypton-Suite/Standard-Toolkit/issues/99
-        public int CornerRadius
-        {
-            get => _checkButton.StateCommon.Border.Rounding;
-
-            set
-            {
-                if (_checkButton.StateCommon.Border.Rounding != value)
-                {
-                    _service.OnComponentChanged(_checkButton, null, _checkButton.StateCommon.Border.Rounding, value);
-
-                    _checkButton.StateCommon.Border.Rounding = value;
-                }
-            }
-        }
         #endregion
 
         #region Public Override
@@ -168,10 +95,6 @@ namespace Krypton.Toolkit
                 actions.Add(new KryptonDesignerActionItem(new DesignerVerb(_action, OnCheckedClick), "Appearance"));
                 actions.Add(new DesignerActionPropertyItem("ButtonStyle", "Style", "Appearance", "Button style"));
                 actions.Add(new DesignerActionPropertyItem("Orientation", "Orientation", "Appearance", "Button orientation"));
-                actions.Add(new DesignerActionPropertyItem("ContextMenuStrip", "Context Menu Strip", "Appearance", "The context menu strip for the control."));
-                actions.Add(new DesignerActionPropertyItem("ShortTextFont", "Short Text Font", "Appearance", "The short text font."));
-                actions.Add(new DesignerActionPropertyItem("LongTextFont", "Long Text Font", "Appearance", "The long text font."));
-                actions.Add(new DesignerActionPropertyItem("CornerRadius", "Corner Rounding Radius", "Appearance", "The corner rounding radius of the control."));
                 actions.Add(new DesignerActionHeaderItem("Values"));
                 actions.Add(new DesignerActionPropertyItem("Text", "Text", "Values", "Button text"));
                 actions.Add(new DesignerActionPropertyItem("ExtraText", "ExtraText", "Values", "Button extra text"));

